src/main/java/marquez/service/mappers/Mapper.java

```diff
@@ -26,46 +28,9 @@
 import javax.annotation.Nullable;
 import lombok.NonNull;
 import marquez.common.Utils;
-import marquez.common.models.DatasetName;
-import marquez.common.models.DatasetType;
-import marquez.common.models.Field;
-import marquez.common.models.FieldType;
-import marquez.common.models.JobName;
-import marquez.common.models.JobType;
-import marquez.common.models.NamespaceName;
-import marquez.common.models.OwnerName;
-import marquez.common.models.SourceName;
-import marquez.common.models.SourceType;
-import marquez.db.models.DatasetFieldRow;
-import marquez.db.models.DatasetRow;
-import marquez.db.models.DatasetVersionRow;
-import marquez.db.models.ExtendedDatasetRow;
-import marquez.db.models.ExtendedRunRow;
-import marquez.db.models.JobContextRow;
-import marquez.db.models.JobRow;
-import marquez.db.models.JobVersionRow;
-import marquez.db.models.NamespaceOwnershipRow;
-import marquez.db.models.NamespaceRow;
-import marquez.db.models.OwnerRow;
-import marquez.db.models.RunArgsRow;
-import marquez.db.models.RunRow;
-import marquez.db.models.RunStateRow;
-import marquez.db.models.SourceRow;
-import marquez.db.models.StreamVersionRow;
-import marquez.service.models.Dataset;
-import marquez.service.models.DatasetMeta;
-import marquez.service.models.DbTable;
-import marquez.service.models.DbTableMeta;
-import marquez.service.models.Job;
-import marquez.service.models.JobMeta;
-import marquez.service.models.Namespace;
-import marquez.service.models.NamespaceMeta;
-import marquez.service.models.Run;
-import marquez.service.models.RunMeta;
-import marquez.service.models.Source;
-import marquez.service.models.SourceMeta;
-import marquez.service.models.Stream;
-import marquez.service.models.StreamMeta;
+import marquez.common.models.*;
+import marquez.db.models.*;
+import marquez.service.models.*;
```

Comment:
_Minor_: We'll want to avoid wildcard imports, see [3.3.1-wildcard-imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
Looks like the  wildcard imports was checked in again in your recent commit?